src/vs/workbench/browser/parts/menubar/menubarPart.ts

```diff
@@ -428,6 +439,12 @@ export class MenubarPart extends Part {
 	}
 
 	private setupCustomMenubar(): void {
+		// Don't update while using the menu
+		if (this.isFocused) {
+			this.menuUpdater.schedule(100);
```

Comment:
@sbatten this seems not very elegant, wouldn't it be smarter to trigger an update when you detect focus is lost instead?

Reply:
good idea, changed